.github/workflows/test-spm.yml

```diff
@@ -0,0 +1,78 @@
+name: Test SPM
+
+on:
+  push:
+    branches: stable
+    paths:
+      - '.github/workflows/test-spm.yml'
+      - '**/*.swift'
+  pull_request:
+    paths:
+      - '.github/workflows/test-spm.yml'
+      - '**/*.swift'
+
+env:
+  xcode-version: 12.0.0
+
+jobs:
+  linux:
+    name: Linux
+    runs-on: ubuntu-latest
+    container: swiftgen/swift:5.3
+    steps:
+      - 
+        name: Checkout
+        uses: actions/checkout@v2
+      - 
+        name: Cache gems
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+      - 
+        name: Cache SPM
+        uses: actions/Cache@v2
```

Comment:
Does it matter that the name of the action we use here is titlecased `Cache` while on line 28 it's all lowercase? Not sure which one is supposed to be the official one but would be nice to be consistent

Reply:
Good catch, didn't notice that (copy paste)